framework/test/mocha/network/setup/config.js

```diff
@@ -52,34 +54,44 @@ const config = {
 		// Generate config objects
 		const configurations = _.range(TOTAL_PEERS).map(index => {
 			const devConfigCopy = _.cloneDeep(devConfig);
-			devConfigCopy.ip = '127.0.0.1';
-			devConfigCopy.wsPort = 5000 + index;
-			devConfigCopy.httpPort = 4000 + index;
-			devConfigCopy.logFileName = `../logs/lisk_node_${index}.log`;
+
+			// Remove dynamic added items in the configuration for tests
```

Comment:
Why is it necessary?

Reply:
It was creating unnecessary large size of config files. These values are added runtime when application load, so no need to be there in config file.